### PR TITLE
Add offender-management-test environment for Manage POM Cases

### PR DESCRIPTION
**Pull request by ayqazi**
## Purpose

This is the first PR for a new environment is to allow us to test WIP code on branches using the same synthetic data as our staging environment. This allows us to test WIP code with richer data than is available on our local machines without requiring security clearance to use live data.

### CODE REVIEW

File: namespaces/live.cloud-platform.service.justice.gov.uk/offender-management-test/00-namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: offender-management-test
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "true"
```
Comment (jackstockley89) [inline namespaces/live.cloud-platform.service.justice.gov.uk/offender-management-test/00-namespace.yaml:6]: could this please be changed to 
`cloud-platform.justice.gov.uk/is-production: "false"`

File: namespaces/live.cloud-platform.service.justice.gov.uk/offender-management-test/resources/variables.tf [hunk 2]
```diff
@@ -0,0 +1,19 @@
+variable "environment-name" {
+  default = "test"
+}
+
+variable "team_name" {
+  default = "offender-management"
+}
+
+variable "is-production" {
```
Comment (jackstockley89) [inline namespaces/live.cloud-platform.service.justice.gov.uk/offender-management-test/resources/variables.tf:9]: could this please be changed to 
`variable "is_production" {
  default = "false"
}`

File: namespaces/live.cloud-platform.service.justice.gov.uk/offender-management-test/00-namespace.yaml (hunk 1 above)
Comment (ayqazi) [reply namespaces/live.cloud-platform.service.justice.gov.uk/offender-management-test/00-namespace.yaml]: addressed

**Review by jackstockley89 — APPROVED** [review]
LGTM

**Comment by ayqazi** [comment]
Converting to draft for 2 reasons:

1) I have to ensure the service is hooked up to the right domain. It's not a standard hosting so I can't just look at the docs
2) It has to connect to the staging DB - the current IaC creates its own DB

**Pull request merged**